PortfolioStyle.css

```diff
@@ -0,0 +1,219 @@
+
+/*Font*/
+
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:ital,wght@0,100;0,200;0,400;1,100;1,200&display=swap');
+	
+/*General CSS - normalize file contains most of it */
+	
+* {
+	box-sizing: border-box;
+}  
+	
+/* Using rems - set up to be 10px = 1 rem*/
+
+body {
+	font-size: 62.5%;
+	font-family: "Josefin Sans";
+	font-weight: 100;
+	background-color: #FCF7F8;
+}
+/* Colours
+
+Background: #FCF7F8 (Pinky White)
+Borders: #CED3DC (Grey)
+Black ish: #000500
+Hover color: #ABA9C3 (Lilac)
+Accents: #88292F (Red)
+
+*/
+
+
+
+/*Text*/
+
+h1 {
+	font-size: 5rem;
+}
+
+h2 {
+	font-size: 2rem;
+	text-align: center;
+	font-weight: 200;
+	padding: 20px 0 0 0;
+}
+
+h3 {
+	font-size: 1.5rem;
+}
+
+h4 {
+	font-size: 1rem;
+	font-weight: 200;
+}
+h6 {
+	font-size: 0.9rem;
+	padding: 20px;
+	font-weight: 100;
+}
+
+p {
+	font-size: 1.2rem;
+	text-align: center;
+}
+
+
+/*Nav Bar*/
+
+nav {
+	overflow: hidden;
+	position:fixed;
+	top: 0;
+	width: 100%;
+	margin: 0;
+	font-size: 1.5rem;
+	background-color: #FCF7F8;
+}
+
+li {
+	float: left;	
+	list-style-type: none;
+	padding: 0 0 20px 0;
+
+}
+
+li a {
+	padding: 8px;
+	text-decoration: none;
+	color: #000500;
+	border-right: 1px solid #000500;
+}
+
+li a:hover {
+	background-color: #ABA9C3;
+}
+
+
+
+/*Body Styling*/
+
+.container {
+	display: flex;
+	height: 100vh;
+	justify-content: space-between;
+	flex-wrap: wrap;
```

Comment:
Depends what you want to achieve here but you might want flex-wrap: row wrap as this will act responsively. But inorder to do this you'll need to create a set size on your box class as currently they are trying to take up 30% of the available space and will always have 3 per row because of this.

Reply:
Thank you, going forward and working on mobile responsivity I will sort this!